l2-frontend/src/construct/ParaclinicResearchEditor.vue

```diff
@@ -512,6 +521,9 @@ export default {
       return undefined;
     });
     this.$root.$on('hide_fte', () => this.f_templates_hide());
+    Object.keys(this.permanent_directories).forEach(key => {
```

Comment:
Такое лучше делать через map

Будет что то вроде

```suggestion
    this.permanent_directories_keys = Object.keys(this.permanent_directories).map(k => ({id: key, label: key}));
```

Reply:
Ну и пустое значение тоже поддержать надо.
Видимо строчкой ниже, можно так:
```js
this.permanent_directories_keys = [{ id: -1, label: 'не выбран' }, ...this.permanent_directories_keys];
```